backend/src/main/java/WebApi.java

```diff
@@ -78,6 +92,20 @@ private static void setupDigitalPartsInterface() {
         put(DIGITALPART + DIGITALPART_ID, (request, response) -> dpi.updateDigitalPart(gson.fromJson(request.body(), DigitalPart.class)), gson::toJson);
     }
 
+    private static void setupPhysicalInterface() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "setupPhysicalInterface" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUnusedPrivateMethod)